src/vm/certificate.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'active_support/core_ext/string/strip'
+
+module Metalware
+  class Vm
+    class Certificate
+      attr_reader :node
+
+      def initialize(node)
+        @node = node
+      end
+
+      def exist?
+        File.exist?(certificate_key_path)
+      end
+
+      def generate
+        generate_certificate_key
+        generate_certificate_info
+        generate_server_certificates
+        puts certificate_generation_message
+        exit
```

Comment:
Don't think you need this line now, with the addition of the `return` in `Vm` the command should exit anyway if the certs need to be generated, and removing this will make this class more reusable for other uses (e.g. maybe later we'll want a command which generates several certs at once, this line would make this impossible as-is as we would exit immediately after generating the first).